pyccl/ccl_core.i

```diff
@@ -4,15 +4,6 @@
 /* put additional #include here */
 %}
 
-// SWIG black magic. Change the behaviour of setting A_SPLINE_MAX to throwing an
-// error.
-%typemap(memberin) double A_SPLINE_MAX {
-    if($input) {
-        PyErr_SetString(PyExc_RuntimeError, "A_SPLINE_MAX is fixed to 1.0 and is not mutable.");
-        SWIG_fail;
-    }
-}
-
```

Comment:
Why did you remove this?

Reply:
I moved it to Python! See `parameters.CCLParameters.__setattr__`.